### PR TITLE
epochLength no longer a governance parameter

### LINKED ISSUE #665: Remove Epochlength from goverance parameters
Take it as a cons param

### PR DESCRIPTION
**Pull request by SamAg19**
fixes #665

### CODE REVIEW

File: contracts/Core/storage/Constants.sol [hunk 1]
```diff
@@ -23,6 +23,9 @@ contract Constants {
     }
 
     uint8 public constant NUM_STATES = 5;
+
+    uint16 public constant EPOCH_LENGTH = 300;
```
[the remark this answers is not in the document]
Comment (SamAg19) [reply contracts/Core/storage/Constants.sol:27]: We can't do deploy as a constructor in state manager as we are just inheriting the state manager and not deploying it individually.  
We may require constructors in the main contracts to deploy it as a constructor param but in this way also we would have to provide epochLength in modifiers. we are saving gas right now by hardcoding the epochLength and reducing the params so i don't see a concrete upside of having it as a constructor param

**Review by abhishekvispute — DISMISSED** [review]
lgtm

**Review by hrishikeshio — DISMISSED** [review]

**Comment by hrishikeshio** [comment]
@SamAg19 please resolve conflicts

**Review by abhishekvispute — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]
lgtm

**Pull request merged**